src/javascript/app/base/header.js

```diff
@@ -361,10 +361,13 @@ const Header = (() => {
             };
 
             const messages = {
-                cashier_locked    : () => localize('Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'),
-                system_maintenance: () => getSystemMaintenanceMessage(),
-                currency          : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]account currency[_2] to enable deposits and withdrawals.'),                                                                                    'user/set-currency'),
-                unsubmitted       : () => buildMessage(get_account_status.risk_classification === 'high'
+                cashier_locked        : () => localize('Your cashier is currently locked. Please contact us via live chat to find out how to unlock it.'),
+                system_maintenance    : () => getSystemMaintenanceMessage(),
+                currency              : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]account currency[_2] to enable deposits and withdrawals.'),                                                                                    'user/set-currency'),
```

Comment:
Please remove these unnecessary spaces between message and link.

Reply:
Orite will do so 🙏

Reply:
Lol